internal/tests/testrepository/header_testrepository.go

```diff
@@ -238,6 +238,18 @@ func (r *HeaderTestRepository) GetMerkleRootsConfirmations(
 	return mrcfs, nil
 }
 
+func (r *HeaderTestRepository) GetHeadersStartHeight(hashtable []string) (int, error) {
+	return 0, nil
+}
+
+func (r *HeaderTestRepository) GetHeadersByHeightRange(from int, to int) ([]*domains.BlockHeader, error) {
+	return nil, nil
+}
+
+func (r *HeaderTestRepository) GetHeadersStopHeight(hashStop string) (int, error) {
+	return 0, nil
+}
+
```

Comment:
maybe it's better to panic here with message "not implemented"

Reply:
implementation added